themes/default/content/blog/amazon-ecr-public/index.md

```diff
@@ -4,7 +4,7 @@ date: "2020-12-01"
 meta_desc: "Pulumi container images now available on Amazon ECR Public"
 meta_image: "pulumi-images-ecr.png"
 authors: ["paul-stack"]
-tags: ["aws", "containers", "ecr"]
+tags: ["aws", "docker-containers"]
```

Comment:
Not sure there's a need to rename `containers` to `Docker-containers`.

Reply:
Fair. Fixed.